go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/metal3-io/cluster-api-provider-metal3
 
-go 1.20
+go 1.22
```

Comment:
Similarly to BMO, go.mod should not be touched by this sort of changes.

Reply:
This is taken care of!